packages/site/landing/index.html

```diff
@@ -25,7 +25,7 @@
             </div>
             <div class="cards">
                 <div class="card-wrapper">
-                    <a class="card" href="storybook/">
+                    <a class="card" href="storybook/nimble">
```

Comment:
early feedback:
- I don't want to break any of our current urls
- Duplicating some config feels like one thing, duplicating lots of source feels like another
- I'm curious if long-term we do actually want separate storybooks

Can you make a meeting to just brainstorm these things with @jattasNI and I?

Reply:
Updated description with new direction that we agreed upon.